docs/guide/alerts-guide.md

```diff
@@ -68,28 +68,19 @@ A **Save Query** window will open with the option to **Enable Alert**. When it's
 
 - **Schedule**, to choose when the Alerts will notify you based on a period, schedule or time.
 
-![Set Alert Schedule](../images/guide/alerts-and-events/save-alert-3.png)
+<video style="display:block; width:100%; height:auto;" autoplay loop="loop">
+  <source src="../../images/guide/alerts-and-events/alert-scheduling.mp4" type="video/mp4" />
+</video>
+
 
 Press Save and you're done. Check out the [integrations](https://sematext.com/docs/integration/) documentation for more information about alert notification hooks.
 
 ### Alerting on Metrics
 
-Metrics have both **Threshold** and **Anomaly** Alerts, but a Monitoring App has **component-based Alerting**. Every time-series metric is a _component_. Alerts can be created for each component, just as they can be added to custom [dashboards](./dashboards-guide/).
+Monitoring Apps have **metric-based Alerting**. Metrics have both **Threshold** and **Anomaly** Alerts which can be created for each metrics component, just as they can be added to custom [dashboards](./dashboards-guide/).
```

Comment:
"Alerts which can be created for each metrics component, just as they can be added to custom dashboards" sounds incorrect.
Alerts in Monitoring Apps are created on specific metrics.  They have nothing to do with components or dashboards.

Reply:
I've re-worded this to be more concrete.